overlord/servicestate/quota_control.go

```diff
@@ -100,6 +262,12 @@ func quotaGroupsAvailable(st *state.State) error {
 	if !enableQuotaGroups {
 		return fmt.Errorf("experimental feature disabled - test it by setting 'experimental.quota-groups' to true")
 	}
+
+	// check if the systemd version is too old
+	if systemdVersion < 205 {
```

Comment:
we should check this before enableQuotaGroups I suppose

Reply:
done